app/src/db/migrations/20250130000000_021-drop-permit-connect.ts

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable max-len */
+import type { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+  return Promise.resolve().then(() =>
+    knex.schema.alterTable('enquiry', (table) => {
+      table.dropColumn('apply_for_permit_connect');
+    })
+  );
+}
+
+export async function down(knex: Knex): Promise<void> {
+  return Promise.resolve()
+
+    .then(() =>
+      knex.schema.alterTable('enquiry', (table) => {
+        table.text('apply_for_permit_connect');
+      })
+    );
+}
```

Comment:
Style: Inconsistent indent formatting when compared with the up function.

Reply:
Style fixed, it slipped through the linter.